website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "琚致远：95 后 Apache Member 的成长之路"
+authors:
+  - name: "饶雅惠"
+    title: "Author"
+    url: "https://github.com/EstelleRao"
+    image_url: "https://github.com/EstelleRao.png"
+keywords:
+- Apache APISIX
+- API 网关
+- ASF Member
+- Apache Member
+- 开源
+- 远程
+- web 全栈工程师
+description: 2022 年 3 月，Apache APISIX PPMC 成员琚致远（GitHub ID：juzhiyuan）入选 Apache Member 和中国开源码力榜。Apache APISIX 社区对此表示祝贺，也邀请了致远做客访谈，聊聊他的开源经历和技术成长之路。
+tags: [Interview]
+---
+
+> 2022 年 3 月，Apache APISIX PPMC 成员琚致远（GitHub ID：juzhiyuan）入选 Apache Member 和中国开源码力榜。Apache APISIX 社区对此  
+表示祝贺，也邀请了致远做客访谈，聊聊他的开源经历和技术成长之路。
+
+<!--truncate-->
+
+## 导语
+
+**琚致远（GitHub ID：juzhiyuan）于 1997 年出生，2019 年 10 月成为 Apache APISIX PPMC Member，2022 年 3 月入选 Apache Member 和  
+中国开源码力榜。他是开源软件商业公司 API7.ai 的 Global Team Leader，也是 ApacheCon Speaker、OSPP Mentor、GSoC Mentor、（2019 年之前）freeCodeCamp 中国核心组织者之一。**
```

Comment:
If this is a paragraph, please do not split it into multiple lines. Please check the full text.